ArmTemplates/CosmosDb/cosmos-collection.md

```diff
@@ -26,11 +26,16 @@ offerThroughput (optional)  int
 
 When provisionRequestUnits is set to true, provisions the RU for the collection
 
-partitionKey (required) string
+partitionKey (optional) string
 
-The partition key to configure the collection to use
+The partition key to configure the collection to use. Leave unset to deploy a partitionless collection.
 
 timeToLive (optional) int
 
 Sets the default time to live for documents within the collection.
-Defaults to -1 (disabled)
+
+If set to null, items are not expired automatically.
```

Comment:
My understanding of the code was if this is set to less the -1 it would not expire (the template would insert null) not that you should set the timeToLive paramater to null